images/goland/Dockerfile.ubuntu

```diff
@@ -1,24 +1,7 @@
-FROM ubuntu:20.04
+FROM codercom/enterprise-multieditor:ubuntu
 
-RUN apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get install -y \
-    git \
-    bash \
-    curl \
-    htop \
-    man \
-    vim \
-    ssh \
-    sudo \
-    lsb-release \
-    ca-certificates \
-    locales \
```

Comment:
@nathanpotter did we need these 3?

Reply:
yeah we do need those :+1:

Reply:
the ca-certs is for successfully downloading extensions, lsb-release is nice to show os-info, and locales I think was a code-server requirement

Reply:
Cool, added these for all 3 in base instead of multieditor, since presumably we'd need it even in non-multieditor images.